recipes/Xform/meta.yaml

```diff
@@ -0,0 +1,69 @@
+{% set name = "xforms" %}
+{% set version = "0.9999.3" %}
+{% set tag = "v0r9999p3" %}
```

Comment:
Is this used at all in the recipe? If not, it should probably be removed.

Reply:
Indeed `tag` is not used, so I have removed it from the meta.yaml.